### PR TITLE
2023 12 1 RNs

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** 2023.12.1 release notes entry
- **Is this PR related to a Monolith release?** Yes 2023.12.1

### Resolves:

* [OKTA-674198](https://oktainc.atlassian.net/browse/OKTA-674198)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/1270c666b79f042d0c0061e55b29dfccc2dd23e8/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 72 | 520 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9588bd4b-fd42-45e2-8545-82367b39ab73) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/1270c666b79f042d0c0061e55b29dfccc2dd23e8/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 74 | 364 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/42fee1b3-4a55-4879-b535-c477598eca4b) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 2 of 2 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.4a98dbc9-4942-4142-877e-561d3c47230c?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 1]
```diff
@@ -8,6 +8,16 @@ title: Okta Identity Engine API Products release notes 2023
...
 
+### Weekly release 2023.12.1
+
+| Change | Expected in Preview Orgs |
+| ------ | ------------------------ |
+| [DPoP support for Okta management APIs is EA in Preview](#dpop-support-for-okta-management-apis-is-ea-in-preview)| December 13, 2023 |
+
+#### DPoP support for Okta management APIs is EA in Preview
+
+You can now use OAuth 2.0 Demonstrating Proof-of-Possession (DPoP) access tokens to access Okta management APIs. See [Configure OAuth 2.0 Demonstrating Proof-of-Possession](/docs/guides/dpop/oktaresourceserver/main/).<!-- OKTA-673922 OKTA_RESOURCE_SERVER_DPOP_SUPPORT-->
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:19]:
```suggestion
You can now use OAuth 2.0 Demonstrating Proof-of-Possession (DPoP) access tokens to access Okta management APIs. See [Configure OAuth 2.0 Demonstrating Proof-of-Possession](/docs/guides/dpop/nonoktaresourceserver/main/).<!-- OKTA-673922 OKTA_RESOURCE_SERVER_DPOP_SUPPORT-->
```

File: packages/@okta/vuepress-site/docs/release-notes/2023/index.md [hunk 2]
```diff
@@ -6,6 +6,16 @@ title: Okta API Products release notes 2023
...
 
+### Weekly release 2023.12.1
+
+| Change | Expected in Preview Orgs |
+| ------ | ------------------------ |
+| [DPoP support for Okta management APIs is EA in Preview](#dpop-support-for-okta-management-apis-is-ea-in-preview)| December 13, 2023 |
+
+#### DPoP support for Okta management APIs is EA in Preview
+
+You can now use OAuth 2.0 Demonstrating Proof-of-Possession (DPoP) access tokens to access Okta management APIs. See [Configure OAuth 2.0 Demonstrating Proof-of-Possession](/docs/guides/dpop/oktaresourceserver/main/).<!-- OKTA-673922 OKTA_RESOURCE_SERVER_DPOP_SUPPORT-->
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023/index.md:17]:
```suggestion
You can now use OAuth 2.0 Demonstrating Proof-of-Possession (DPoP) access tokens to access Okta management APIs. See [Configure OAuth 2.0 Demonstrating Proof-of-Possession](/docs/guides/dpop/nonoktaresourceserver/main/).<!-- OKTA-673922 OKTA_RESOURCE_SERVER_DPOP_SUPPORT-->
```

**Review by susanharper-okta — COMMENTED** [review]
url

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/d25a8d233e8a1f0b6f324f99a89b208d8272af2e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 72 | 520 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f0ab0605-4223-494f-bae1-cf899671a4e4) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/d25a8d233e8a1f0b6f324f99a89b208d8272af2e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 74 | 364 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3eb1c9a1-efae-4e5b-8037-a719b7ae5979) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 2 of 2 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.0ee4254d-ade1-4e4d-ab6a-432e31d93aed?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/5d76338150f5a1757523c7239e3e09a70d434614/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 72 | 520 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ec90fbfd-fd81-4b44-b288-e326d6c78ff4) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/5d76338150f5a1757523c7239e3e09a70d434614/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 74 | 364 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7c50606a-31f4-433d-98f8-917f3b35598c) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 2 of 2 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.a714faf9-c103-403d-b69c-8bf8f8fc457d?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Pull request merged**